content/zh/docs/tasks/security/authentication/_index.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Authentication
-description: Controlling mutual TLS and end-user authentication for mesh services.
+title: 认证
```

Comment:
Please don't mix Chinese translation with other change in the same PR. Translation should be done in PRs with the `translation/chinese` label set.